projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -1236,6 +1236,7 @@ s_archive
          | LOG
```

Comment:
This innermost parenthetical expression should be moved to a new rule `a_null` or something similar if that is already taken.